tools/mig-test/README.md

```diff
@@ -0,0 +1,36 @@
+# Flink CDC Migration Test Utilities
+
+## Pipeline Jobs
+### Preparation
+
+1. Install Ruby (macOS has embedded it by default)
+2. (Optional) Run `gem install terminal-table` for better display
+
+### Compile snapshot CDC versions
+3. Set `CDC_SOURCE_HOME` to the root directory of the Flink CDC git repository
+4. Run `ruby prepare_libs.rb` to download released / compile snapshot CDC versions
+
+### Run migration tests
+5. Enter `conf/` and run `docker compose up -d` to start up test containers
+6. Set `FLINK_HOME` to the home directory of Flink
+7. Run `ruby run_migration_test.rb` to start testing
```

Comment:
Setting `FLINK_HOME` could not be automated since one must specify it somewhere. Maybe leaving `docker compose up` separately might be better so we don't have to restart docker containers every time.

Reply:
Considering that this is mainly provided for developers and there are not many steps involved, I can also accept the current mode.